jlibra-core/src/main/java/dev/jlibra/serialization/Serializer.java

```diff
@@ -25,6 +25,10 @@ public Serializer appendByteArray(byte[] byteArray) {
                 .append(byteArray);
     }
 
+    public Serializer appendByteArrayWithoutLengthInformation(byte[] byteArray) {
```

Comment:
We could have these 2 functions the other way around. If `appendByteArray` only appends array but no length then we could have `appendLength`. Then this one is not needed (I don't like long names) and instead we can use 
`appendLength(b).appendByteArray(b)` and just `appendByteArray(b)`?

Reply:
I see your point, but the reason why I chose to do it like this was that this is an exception in the serialization. Normally, and in all other cases the length information is added before the actual data, but now they have decided to do it differently for the address data. I wouldn't want to remember to add the length explicitly when doing serializations because in all other cases it is required.